tests/common.go

```diff
@@ -6624,3 +6625,123 @@ func AddMetadataDisk(n node.Node) error {
 	return nil
 
 }
+
+// generateRandomString Creates a random string of length n and returns it
+func generateRandomString(n int) string {
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	randomString := make([]rune, n)
+	for i := range randomString {
+		randomString[i] = letters[rand.Intn(len(letters))]
```

Comment:
minor optimization, len(letters) can be cached outside loop to avoid it calling repeatedly.

Reply:
No longer needed as I am using the existing `RandomString()` function